src/20240115_Multi_HarmonizeUSDTRiskParametersOnAaveV3Markets/HarmonizeUSDTRiskParametersOnAaveV3Markets.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Harmonize USDT Risk Parameters on Aave V3 Markets"
+author: "Aave Chan Initiative"
+discussions: "https://governance.aave.com/t/arfc-harmonize-usdt-risk-parameters-on-aave-v3-markets/15815"
+---
+
+## Simple Summary
+
+USDT has various risk parameters across the various Aave v3 markets. This proposals aims to harmonize the risk parameters to better align the markets by removing USDT from isolation mode and normalizing risk parameters.
+
+## Motivation
+
+USDT has seen strong borrow demand across networks with borrow utilization over 50% on Aave v3 Ethereum Mainnet and good utilization on secondary networks. However, the various markets currently have different risk parameters, leading to some inefficiency.
+
+This proposal aims to bring the various USDT markets in line with a single set of LTV, liquidation threshold, and liquidation bonus parameters.
+
+## Specification
+
+The proposal sets the LT/LTV of USDT on all deployments at 80/77, matching that of DAI and USDC on v3 Ethereum. Regarding the LB, it will stay the same at 5% on the L2s and 4.5% on Ethereum.
```

Comment:
I would also put a note that once an asset is removed from isolation mode, it could not be added back